docs/reference/contributing/guidelines/workflow.md

```diff
@@ -36,31 +44,35 @@ Please see the [code contributions chapter](/docs/v5.7/reference/guidelines.html
 
 Pull requests on GitHub have to meet the following requirements to keep the code and commit history clean:
 
-- Commits should always contain a proper description of their content. Start with a concise and sensible one-line description. Then, elaborate on reasoning of the choices made, descriptions for reviewers and other information that might otherwise be lost.
+- Commits should always contain a proper description of their content. Start with a concise and sensible one-line description. Then, elaborate on reasoning of the choices made, descriptions for reviewers and other information that might otherwise be lost.
 - You should always write commits to allow publication, so they can never contain confidential information, reference private documents, links to intranet locations or rude language.
 - Each commit should be the minimum self-contained commit for a change. A commit should always result in a new state that is again in a compilable state. You should (if possible) split large changes into logical smaller commits that help reviewers follow the reasoning behind the full change.
 - Commits should follow [Chris Beam’s seven rules of great commit messages](http://chris.beams.io/posts/git-commit#seven-rules):
-	1. Separate subject from body with a blank line.
-	1. Limit the subject line to 72 characters (note that this is a deviation from Beam's standard).
-	1. Capitalize the subject line.
-	1. Do not end the subject line with a period.
-	1. Use the imperative mood in the subject line.
-	1. Wrap the body at 72 characters.
-	1. Use the body to explain _what_ and _why_ vs _how_.
+    1. Separate subject from body with a blank line.
+    1. Limit the subject line to 72 characters (note that this is a deviation from Beam's standard).
+    1. Capitalize the subject line.
+    1. Do not end the subject line with a period.
+    1. Use the imperative mood in the subject line.
+    1. Wrap the body at 72 characters.
+    1. Use the body to explain _what_ and _why_ vs _how_.
```

Comment:
Why the spaces? The indentation should be handled automatically by the markdown tags. Though looks like you have lost those for some reason hence the '1.' everywhere ?

Reply:
yes will revert this somehow got messed during the updates